### ISSUE TITLE
Errore in fotogrammetria

### ISSUE THREAD

**Issue by Guisan** [cogitas3d/LiberTeeth3D#1]
Durante il processo di fotogrammetria compare questo errore. Allego il testo della Toggle System Console.
![screenshot 184](https://user-images.githubusercontent.com/17667365/37465321-21224366-285b-11e8-806f-16b4e0bbcd82.png)

[ErrorLiberTeeth.txt](https://github.com/cogitas3d/LiberTeeth3D/files/1815493/ErrorLiberTeeth.txt)

**Comment by cogitas3d** [comment]
Hi @Guisan !

Did you install OrtogOnBlender? I update the installer with support to LiberTeeth3D:

https://drive.google.com/file/d/1rMS9yuNxan7dVfmnjxdbb9qFnPOUEFRk/view?usp=sharing

Inside it already have photos and CT-Scan to test (~/ARQUIVOS/ARCADAS).

Please, try and tell me whats happen.

A big hug!

**Comment by Guisan** [comment]
Yes, I've installed OrtogOnBlender with support to LiberTeeth3D on Windows 10 64 bit. I've used for the test (~/ARQUIVOS/ARCADAS) but the photogrammetry has given the error I've posted above. 
Cheers!
Guido Santandrea
